### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by lbussell**
A couple of changes:
* Document our current support policy for .NET feature updates
* Remove bootstrap workaround instructions (no longer necessary)

### CODE REVIEW

**Review by omajid — APPROVED** [review]
Thanks!

File: README.md [hunk 1]
```diff
@@ -39,22 +39,6 @@ The steps to build vary slightly depending on your distro.  The following set of
 
     This downloads a .NET SDK and a number of .NET packages needed to build .NET from source.
 
-    > On Linux distros other than Fedora 33, an additional bootstrapping step is required.  After running `prep.sh` above, run the following:
```
Comment (MichaelSimons) [inline README.md:42]: I believe the bootstrap option is still necessary for building on arm64

File: README.md [hunk 2]
```diff
@@ -104,6 +88,15 @@ Source-build solves common challenges that most developers encounter when trying
...
 
+
+## Support
+
+.NET Source-Build is supported on the oldest available .NET SDK feature update.
+For example, if both .NET 6.0.1XX and 6.0.2XX feature updates are available from [dotnet.microsoft.com](https://dotnet.microsoft.com/en-us/download/dotnet/6.0), Source-Build will support 6.0.1XX.
+For the latest information about Source-Build support for new .NET versions, please check our [GitHub Discussions page](https://github.com/dotnet/source-build/discussions) for announcements.
+
+* [More information about .NET Versioning](https://docs.microsoft.com/en-us/dotnet/core/versions/)
+* [More information about .NET Support Policies](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core)
```
Comment (MichaelSimons) [inline README.md:99]:
```suggestion
* [More information about .NET Support Policies](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core)

```

File: README.md [hunk 3]
```diff
@@ -104,6 +88,15 @@ Source-build solves common challenges that most developers encounter when trying
 
 Starting with .NET 6, the core source-build infrastructure is integrated into the [dotnet/installer](https://github.com/dotnet/installer/tree/main/src/SourceBuild) repo. The `main` branch on this repo now contains the tooling needed to build .NET's external dependencies from source.
 
+
```
Comment (MichaelSimons) [inline README.md:91]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (MichaelSimons) [reply README.md:42]: This issue would eliminate the need for the bootstrap option for arm64 - https://github.com/dotnet/source-build/issues/2758

**Review by MichaelSimons — APPROVED** [review]

**Review by crummel — APPROVED** [review]

**Pull request merged**